File: Selenium/Project/Activity8.py
import time

from selenium import webdriver
from selenium.webdriver.common.by import By

driver = webdriver.Firefox()
driver.get("https://alchemy.hguy.co/lms")
driver.find_element(By.XPATH, "//a[contains(text(),'Contact')]").click()
driver.find_element(By.XPATH,"//form/div/div/label[contains(text(),'Full Name')]/following-sibling::input").send_keys("Someone")
driver.find_element(By.XPATH,"//form/div/div/label[contains(text(),'Email')]/following-sibling::input").send_keys("some@s.com")
driver.find_element(By.XPATH,"//form/div/div/label[contains(text(),'Subject')]/following-sibling::input").send_keys("Subject")
driver.find_element(By.XPATH, "//form/div/div/label[contains(text(),'Comment or Message ')]/following-sibling::textarea").send_keys("Feedback")
driver.find_element(By.XPATH, "//button[@type='submit']").click()

confirmation_message = driver.find_element(By.XPATH, "//div[@id='wpforms-confirmation-8']/p").text
print(confirmation_message)

driver.quit()

File: Selenium/Project/Activity8.java
package Activities;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import io.github.bonigarcia.wdm.WebDriverManager;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.openqa.selenium.interactions.Actions;
import org.openqa.selenium.support.ui.WebDriverWait;
import org.testng.annotations.BeforeTest;
import org.testng.annotations.AfterTest;
import org.testng.annotations.Test;

import java.time.Duration;
import java.util.List;

import static org.testng.Assert.assertEquals;
import static org.testng.Assert.assertTrue;


public class Activity8{
    WebDriver driver;
    WebDriverWait wait;
    @BeforeTest
    public void setUp()
    {
        WebDriverManager.firefoxdriver().setup();
        driver = new FirefoxDriver();
        System.setProperty(FirefoxDriver.SystemProperty.BROWSER_LOGFILE, "/dev/null");
        wait = new WebDriverWait(driver, Duration.ofSeconds(10));
        driver.get("https://alchemy.hguy.co/lms");
    }

    @Test
    public void testOpen()
    {
        driver.findElement(By.linkText("Contact")).click();
        assertEquals(driver.getTitle(), "Contact – Alchemy LMS");
        driver.findElement(By.id("wpforms-8-field_0")).sendKeys("Full Name");
        driver.findElement(By.id("wpforms-8-field_1")).sendKeys("fullname.xyz@abc.com");
        driver.findElement(By.id("wpforms-8-field_2")).sendKeys("Good work!");
        driver.findElement(By.id("wpforms-8-field_3")).sendKeys("Appreciation subject");
        driver.findElement(By.id("wpforms-submit-8")).click();
        System.out.println("Confirmation message: " + driver.findElement(By.id("wpforms-confirmation-8")).getText());
    }

    @AfterTest
    public void cleanUp()
    {
        driver.quit();
    }


}
